designer/server/src/main/scala/pl/touk/nussknacker/ui/api/description/StatisticsApiEndpoints.scala

```diff
@@ -73,6 +84,72 @@ object StatisticsApiEndpoints {
     @derive(encoder, decoder, schema)
     final case class StatisticUrlResponseDto private (urls: List[URL])
 
+    @derive(encoder, decoder, schema)
+    final case class RegisterStatisticsRequestDto private (statistics: List[StatisticDto])
+
+    @derive(encoder, decoder, schema)
+    final case class StatisticDto private (name: StatisticName)
+
+    sealed trait StatisticName extends EnumEntry with UpperSnakecase
+
+    object StatisticName extends Enum[StatisticName] with CirceEnum[StatisticName] {
```

Comment:
the name of the statistic is part of our API now. Changing the name implicates changing the API (the compatibility break). Maybe we should have a test that shows us that someone changed the name of any of these objects (and as a result, they changed the API)?

I see two solutions:
1. business spec in which we use every possible statistic name (in the future, new statistic names won't be included and the developer who is going to extend them, probably will miss adding them to the test)
2. `NuDesignerApiAvailableToExposeYamlSpec` - we can add a test which gets all names from the yaml and checks if the set looks exactly as we want it to look like

Reply:
Done